homeassistant/components/zwave_js/services.py

```diff
@@ -112,7 +112,7 @@ def raise_exceptions_from_results(
 
 
 async def _async_invoke_cc_api(
-    nodes_or_endpoints: set[T],
+    nodes_or_endpoints: Collection[_NodeOrEndpointType],
```

Comment:
The function accept both lists and sets. `Collection` is a common / immutable base class for both, thus making it covariant. _The code never modifies `nodes_or_endpoints` directly, only creates a list from it._